src/precice/tests/explicit-access-direct-mapping.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<precice-configuration>
+  <solver-interface dimensions="2">
+    <data:scalar name="Velocities" />
+    <data:scalar name="Forces" />
+
+    <mesh name="MeshOne">
+      <use-data name="Forces" />
+    </mesh>
+
+    <mesh name="MeshTwo">
+      <use-data name="Velocities" />
+      <use-data name="Forces" />
+    </mesh>
+
+    <participant name="SolverOne">
+      <use-mesh name="MeshOne" provide="yes" />
+      <use-mesh name="MeshTwo" from="SolverTwo" safety-factor="0" direct-access="true" />
```

Comment:
```suggestion
      <use-mesh name="MeshTwo" from="SolverTwo" direct-access="true" />
```
See code discussion above.

Reply:
The test would fail if I change this line. Why should I change it?

Reply:
OK, the safety factor belongs to the mesh below.
Just to be sure: the direct access does not use the safety factor, right?

Reply:
It belongs more or less to the NN mapping below. The direct access does not use the safety factor, correct.